gencert-private.sh

```diff
@@ -91,9 +114,9 @@ EOF
 
 cat <<EOF
 ## --------------------------------------------------------------------------
-## Extract hash in sha256 from private key
+## Extract hash in sha256 from csr
```

Comment:
You're not extracting anything. You're just computing the digest of the CSR file